modules/gitops-disable-dex-using-spec-sso.adoc

```diff
@@ -0,0 +1,25 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-using-dex
+
+:_content-type: PROCEDURE
+[id="gitops-disable-dex-using-spec-sso_{context}"]
+= Enabling/Disabling Dex using .spec.sso
+
+In {gitops-title} v1.6.0 `.spec.dex` is deprecated and support will be removed in v1.9.0. Dex can be configured through `.spec.sso.dex` parameter.
+
+.Procedure
+
+* To enable Dex, set the `.spec.sso.provider: dex` parameter in the *YAML* resource of the Operator:
+
++
+[source,yaml]
+----
+spec:
```

Comment:
- The indentation here is different from that in the other module. Please verify the indentation of both YAML examples.
- Also, please use ellipses to indicate truncated code in code blocks:
See "For snippets or sections of a file, use an ellipsis (…​) to show that the file continues before and/or after the quoted block." in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc